task/prepare.go

```diff
@@ -0,0 +1,121 @@
+package task
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/golang/glog"
+	livepeerAPI "github.com/livepeer/go-api-client"
+	"github.com/livepeer/stream-tester/model"
+	"github.com/livepeer/stream-tester/segmenter"
+)
+
+var profile240p = livepeerAPI.Profile{
+	Name:    "240p0",
+	Fps:     0,
+	Bitrate: 250000,
+	Width:   426,
+	Height:  240,
+	Gop:     "2.0",
+}
+
+var allProfiles = []livepeerAPI.Profile{
+	profile240p,
+	{
+		Name:    "360p0",
+		Fps:     0,
+		Bitrate: 800000,
+		Width:   640,
+		Height:  360,
+		Gop:     "2.0",
+	},
+	{
+		Name:    "480p0",
+		Fps:     0,
+		Bitrate: 1600000,
+		Width:   854,
+		Height:  480,
+		Gop:     "2.0",
+	},
+	{
+		Name:    "720p0",
+		Fps:     0,
+		Bitrate: 3000000,
+		Width:   1280,
+		Height:  720,
+		Gop:     "2.0",
+	},
+}
+
+func RecordStream(ctx context.Context, lapi *livepeerAPI.Client, assetSpec *livepeerAPI.AssetSpec, file io.ReadSeekCloser) (string, error) {
+	streamName := fmt.Sprintf("vod_hls_recording_%s", time.Now().Format("2006-01-02T15:04:05Z07:00"))
```

Comment:
Maybe it will be more useful to have the asset ID instead of a timestamp here?
```suggestion
	streamName := fmt.Sprintf("vod_hls_recording_%s", assetId))
```
(just gotta receive the ID arg)

Actually, this should just receive a full `TaskContext` which will have the `ctx`, `lapi` and the `AssetId` already. This is a crucial part of a task, which could even be a task itself, so I think we should make it have a similar interface.